tests/Installer/MoodleAppInstallerTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Moodle Plugin CI package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Copyright (c) 2018 Blackboard Inc. (http://www.blackboard.com)
+ * License http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace MoodlePluginCI\Tests\Installer;
+
+use MoodlePluginCI\Installer\MoodleAppInstaller;
+use MoodlePluginCI\Tests\Fake\Process\DummyExecute;
+use MoodlePluginCI\Tests\MoodleTestCase;
+
+class MoodleAppInstallerTest extends MoodleTestCase
+{
+    public function testInstall()
```

Comment:
Maybe here we could add some more tests, setting up different configuration options (MOODLE_APP, other globals...). Just towards getting some of those logical branches in the AppInstaller covered.

I also would add some tests to the BehatCommand tests to verify that the new options are processed ok (when defined).

Reply:
I'm ok. We can always add missing coverage of logical branches later. Resolved.